src/reanimated2/InstallFunctions.js

```diff
@@ -87,12 +96,63 @@ export function installFunctions(innerNativeModule) {
     }
     _log(stringRepresentation(data))
   })
+
+  // clamp
+  const clamp = function (x, values) {
+    'worklet';
+    const diffs = values.map((it) => Math.abs(it - x));
+    const index = diffs.indexOf(Math.min(...diffs));
+    return values[index];
+  }
+
+  global.Reanimated.clamp = clamp;
+  install('Reanimated.clamp', clamp);
+
+  // interpolate
+  const interpolate = function (x, l, r, ll, rr, type) {
```

Comment:
what about interpolate with more than 2 args?

```
opacity: interpolate(scrollPosition, {
  input: [0, 0, 1]
  output: [-400, 0, 100] // this is common thing to avoid overscroll
});
```

Reply:
Done :)